docs/utils.rst

```diff
@@ -54,12 +57,17 @@ be explored by selecting which toolkit(s) are and are not registered.
 
 .. code-block:: python
 
-    from openforcefield.utils.toolkits import ToolkitRegistry, OpenEyeToolkitWrapper, RDKitToolkitWrapper
-    toolkit_registry = ToolkitRegistry(toolkit_precedence=[OpenEyeToolkitWrapper, RDKitToolkitWrapper])
-    smiles_via_openeye = toolkit_registry.call('to_smiles', molecule)
-    toolkit_registry.deregister_toolkit(OpenEyeToolkitWrapper)
-    smiles_via_rdkit = toolkit_registry.call('to_smiles', molecule)
-
+    >>> from openforcefield.utils.toolkits import OpenEyeToolkitWrapper, GLOBAL_TOOLKIT_REGISTRY as toolkit_registry
+    >>> from openforcefield.topology.molecule import Molecule
+    >>> molecule = Molecule.from_smiles('Cc1ccccc1')
+    >>> smiles_via_openeye = toolkit_registry.call('to_smiles', molecule)
+    >>> print(smiles_via_openeye)
+    [H]c1c(c(c(c(c1[H])[H])C([H])([H])[H])[H])[H]
+
+    >>> toolkit_registry.deregister_toolkit(OpenEyeToolkitWrapper)
+    >>> smiles_via_rdkit = toolkit_registry.call('to_smiles', molecule)
+    >>> print(smiles_via_rdkit)
+    [H][c]1[c]([H])[c]([H])[c]([C]([H])([H])[H])[c]([H])[c]1[H]
```

Comment:
Could potentially add 
```
>>> Molecule.from_smiles(smiles_via_openeye) == Molecule.from_smiles(smiles_via_rdkit)
True
```
if the difference in how these SMILES looks is a concern